expression/builtin/string.go

```diff
@@ -147,6 +147,23 @@ func builtinUpper(args []interface{}, ctx map[interface{}]interface{}) (interfac
 	}
 }
 
+// See: https://dev.mysql.com/doc/refman/5.7/en/string-comparison-functions.html
+func builtinStrcmp(args []interface{}, ctx map[interface{}]interface{}) (interface{}, error) {
+	if args[0] == nil || args[1] == nil {
+		return nil, nil
```

Comment:
In this case I find that MyQSL will return the error about `Unknown column 'nil' in 'field list'`. Do we need return an error ？